tools/incubator.js

```diff
@@ -69,9 +69,14 @@ const Incubator = new Lang.Class({
     _import_article: function (metadata) {
         let basin_metadata = {};
 
+        let discoveryFeedContent = metadata['discoveryFeedContent'] || {}
+        let tags = ['EknArticleObject'].concat([
```

Comment:
I think the readability of this could be improved:
```js
let tags = metadata['tags'].slice();
tags.unshift('EknArticleObject');
if (discoveryFeedContent['blurbs'])
    tags.unshift('EknHasDiscoveryFeedTitle');
```

Reply:
I'm more partial towards the functional style, but I'm happy to do it this way too

Reply:
I also prefer functional style, but in this case I had to stare at it for a long time before I had any idea what it did :sweat_smile: